helios-ts/src/lib.rs

```diff
@@ -21,7 +24,7 @@ macro_rules! log {
 
 #[wasm_bindgen]
 pub struct Client {
```

Comment:
Lets come up with a way to allow the typescript to set what type of db we want to use here. Maybe we can do something with generics if we can get js and ts to interop with them well?

Reply:
wasm newbie here😅, still learning how to do this. it seems it's easier to just pass 'config' or 'localstorage' to `Client`
```ts
new Client(executionRpc, consensusRpc, network, checkpoint, db || 'localstorage');
```

Reply:
If I use generic in `pub struct Client`, it gives an error:

```
structs with #[wasm_bindgen] cannot have lifetime or type parameters currently
```

So I changed to use an enum